src/components/TasksListItem/TasksListItem.tsx

```diff
@@ -68,19 +75,22 @@ export const TasksListItem = ({
           <div className='Layer__tasks-list-item__body-info'>
             <Text size={TextSize.sm}>{task.question}</Text>
             <Textarea
-              disabled={task.status === 'COMPLETED'}
-              placeholder={userResponse}
+              value={userResponse}
               onChange={(e: React.ChangeEvent<HTMLTextAreaElement>) =>
                 setUserResponse(e.target.value)
               }
             />
             <div className='Layer__tasks-list-item__actions'>
               <Button
                 disabled={
-                  task.status === 'COMPLETED' || userResponse.length === 0
+                  userResponse.length === 0 ||
+                  userResponse === task.user_response
```

Comment:
In addition to disabling the button for empty answers, this makes button disabled if the text hasn't changed from what we think the previous answer is. The effect is that when you open a completed task, the button is disabled until you change the text, then after submitting an update, the button goes back to disabled to help indicate that your answer was saved.